pkg/netpol/connlist/conns_formatter.go

```diff
@@ -68,20 +68,41 @@ func formExposureItemAsSingleConnFiled(peerStr string, exposureItem XgressExposu
 	if exposureItem.IsExposedToEntireCluster() {
 		return formSingleExposureConn(peerStr, entireCluster, exposureItem.PotentialConnectivity(), isIngress)
 	}
-	if len(exposureItem.NamespaceLabels()) > 0 {
-		return formSingleExposureConn(peerStr, peerStrWithNsLabels(exposureItem.NamespaceLabels()),
-			exposureItem.PotentialConnectivity(), isIngress)
+	repPeerStr := writeNsLabels(exposureItem.NamespaceLabels())
```

Comment:
rename func `writeNsLabels`, as the name is misleading (it does not write..),
could be `getRepresentativeNamespaceString()` for example

Reply:
done